sources/packages/api/src/database/migrations/1622595885302-UpdateUserIsActive.ts

```diff
@@ -0,0 +1,20 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
By the format of these files and the black lines along with the code, I assume that they are not formatted.
Can you please check if the "format on save" is configure on VS code? It is supposed to get rid of the blank lines and keep the formatting consistency between devs.